app/Console/Commands/EditImagesCommand.php

```diff
@@ -45,12 +46,17 @@ public function handle(AWS $aws)
         Post::where('id', '>', $start)->chunk(100, function ($posts) use ($aws) {
             foreach ($posts as $post) {
                 $image = $post->url;
```

Comment:
That's a good thought! I wonder how many images we have with `default` as the URL, and what we should do with them. I pushed b9beb26 which prints the URL as well so we can easily see if a ton of these errors are happening because of `default` strings.